src/main/webapp/location/bbox.tsx

```diff
@@ -1,23 +1,11 @@
 import * as React from 'react'
-import { geometry, coordinates as coordinateEditor } from 'geospatialdraw'
-import { Filter, INTERSECTS, ANY_GEO, GEOMETRY } from './filter'
-import { geoToWKT } from './geo-to-wkt'
+import { coordinates as coordinateEditor } from 'geospatialdraw'
 import FormControl from '@material-ui/core/FormControl'
 import FormLabel from '@material-ui/core/FormLabel'
 import Point from './point'
 import Props from './geo-editor'
 import SpacedLinearContainer from '../spaced-linear-container'
 
-export const generateFilter = (geo: geometry.GeometryJSON): Filter => ({
```

Comment:
Why is this being removed?

Reply:
This is no longer needed, filters can be generated directly from GeometryJSON.

Reply:
So the components speak geojson directly? :+1: